packages/dds/matrix/src/permutationvector.ts

```diff
@@ -346,7 +346,7 @@ export class PermutationVector extends Client {
             }
 
             default:
-                assert.fail();
+                throw new Error();
```

Comment:
Having at least some text here would be nice :)

Reply:
Any suggestions for what it should say?